index.js

```diff
@@ -1,38 +1,80 @@
 const HOME_MENU_BUTTONS_ARRAY = [
     {
-        id: "internet-contents",
-        innerText: "Internet Contents"
+        id: "internetcontents-tab",
+        innerText: "Internet Contents",
+        onclick: () => renderInternetContents()
     },
     {
-        id: "digital-programme-list",
-        innerText: "Digital Programme List"
+        id: "digital-programme-list-tab",
+        innerText: "Digital Programme List",
+        onclick: () => renderDigitalProgrammeList()
     },
     {
-        id: "digital-epg",
+        id: "digital-epg-tab",
         innerText: "Digital EPG"
     },
     {
-        id: "applications",
+        id: "applications-tab",
         innerText: "Applications"
     },
     {
-        id: "recordings",
+        id: "recordings-tab",
         innerText: "Recordings"
     },
     {
-        id: "media",
+        id: "media-tab",
         innerText: "Media"
     },
     {
-        id: "settings",
+        id: "settings-tab",
         innerText: "Settings"
     },
 ]
 
+const INTERNET_CONTENS_BUTTONS_ARRAY = [
+    {
+        id: "amazon-prime-video",
+        innerText: "Amazon Prime Video",
+        onclick: () => console.log("Amazon Prime Video")
+    },
+    {
+        id: "netflix",
+        innerText: "Netflix",
+        onclick: () => console.log("Netflix")
+    },
+    {
+        id: "bbc-iplayer",
+        innerText: "BBC iPlayer",
+        onclick: () => console.log("BBC iPlayer")
+    },
+    {
+        id: "youtube",
+        innerText: "YouTube",
+        onclick: () => console.log("YouTube")
+    },
+]
+
+// ELEMENT_IDS
+ELEMENT_IDS = {
+    app: "app",
+    wrapper: "wrapper",
+    internetContentsWrapper: "internet-contents-wrapper-element",
+    internetContentsTab: "internetcontents-tab",
+    programmeListTab: "digital-programme-list-tab",
+    digitalEpgTab: "digital-epg-tab",
+    applicationsTab: "applications-tab",
+    recordingsTab: "recordings-tab",
+    mediaTab: "media-tab",
+    settingsTab: "settings-tab",
```

Comment:
Strange that `settings-tab` string is hardcoded twice in your code

Reply:
And it is not `settings-tab` only

Reply:
Solved